src/client/debugger/configProviders/simpleProvider.ts

```diff
@@ -0,0 +1,66 @@
+import * as path from 'path';
```

Comment:
I assume other providers are going to end up here, hence the generic name?

Reply:
Will rename, got some more information about this on vscode issue i raised.

Reply:
Actually, ignore me; missed that the file is in the `debugger/` sub-directory.